src/components/fullscreen-flyout/OverlayHeader.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+import * as React from 'react';
+
+import './OverlayHeader.scss';
+import classNames from 'classnames';
+import { bdlGray62 } from '../../styles/variables';
+import Button from '../button';
+import IconClose from '../../icons/general/IconClose';
+
+export type ResponsiveMenuProps = {
+    /** Custom primary action button that appears on the right header */
+    actionButton?: React.Element<any>,
+    /** Set className to the overlay header */
+    className?: string,
+    /** Set click handler when close button is clicked */
+    onCloseClick?: Function,
+    /** Primary title in header */
+    primaryTitle?: string,
+    /** Reverse orientation of primary and secondary titles in header */
+    reverseTitle?: boolean,
+    /** Secondary title in header */
+    secondaryTitle?: string,
+};
+
+type CloseButtonProps = {
+    onClick?: Function,
+};
+
+const CloseButton = ({ onClick }: CloseButtonProps) => {
+    return (
+        <Button onClick={onClick} className="close-btn" type="button">
+            <IconClose color={bdlGray62} height={18} width={18} />
```

Comment:
Do we have button size constants we could use for height and width?

Reply:
Not sure if it's appropriate here because this height/width is specific to this button